GrandCentralBoard/Widgets/Harvest/HarvestWidgetSettings.swift

```diff
@@ -25,6 +26,7 @@ struct HarvestWidgetSettings: Decodable {
                                          clientID:        json => "clientID",
                                          clientSecret:    json => "clientSecret",
                                          numberOfDays:    json => "numberOfDays",
+                                         includeWeekends: json =>? "includeWeekends",
```

Comment:
We had this discussion before with @krris. The above is the convention we started in Bonus and Image widgets.

Although I get your point and it's valid, our motivation was:
- to have only "parsing" responsibility in `*Settings` classes (this way, they're easier for test for different edge cases + no need to maintain default values in tests);
- to put "default" values logic in `WidgetBuilding`, since it receives the config json and configures a widget components, so it sounds like a correct place.

I have no strong feelings one way or the other, it's just a consequence of the choice we made and already implemented in few widgets.